### DISCUSSION TITLE
More failsafe Vermin results parsing

### DISCUSSION THREAD

**Discussion by netromdk** [matthewdeanmartin/so_pip#46] [Ideas]
Hello there. I saw that you're using my tool, [vermin](https://github.com/netromdk/vermin). :)

There is another way of parsing the results: you can use the "parsable" format.
For example:
```
% ./vermin.py -f parsable /path/to/examples
/path/to/examples/abc.py:1:7:2.6:3.0:'abc' module
/path/to/examples/abc.py:2::!2:3.4:'abc.ABC' member
/path/to/examples/abc.py:::!2:3.4:
/path/to/examples/except_star.py:::~2:~3:
/path/to/examples/annotations.py:::2.0:3.0:print(expr)
/path/to/examples/annotations.py:1::!2:3.0:annotations
/path/to/examples/annotations.py:::!2:3.0:
:::!2:3.4:
```
The last line will always show the results of the entire scan: `:::`, the py2 version, `:`,  the py3 version, and `:`. Note that for each file, there is always a last entry with `:::` and ending with `:` followed by nothing, which means the computed minimum versions for that file.

`-f parsable` could be added here: https://github.com/matthewdeanmartin/so_pip/blob/cbb18be3900928451fb897eab24d041fdf4c9aaa/so_pip/cli_clients/external_commands.py#L229

And then change the parsing here: https://github.com/matthewdeanmartin/so_pip/blob/cbb18be3900928451fb897eab24d041fdf4c9aaa/so_pip/parse_python/python_validator.py#L24-L42

**Comment by matthewdeanmartin** [comment]
Oh that's cool! I'm going to try to merge that in this weekend, thanks!

**Reply by netromdk** [reply]
You're welcome. :)
